scripts/runtime-upgrade/upgrade_chain.sh

```diff
@@ -0,0 +1,5 @@
+#! /bin/bash
+SCRIPT_DIR="$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
+(cd $SCRIPT_DIR../../ ; cargo build --release)
```

Comment:
Consider making a `ROOT_DIR` variable since you use `$SCRIPT_DIR/../../` in multiple places. Less likely to miss updating one if moving this file.